lib/pf/web/dispatcher.pm

```diff
@@ -88,6 +88,7 @@ sub handler {
     # See L<pf::web::constants::CAPTIVE_PORTAL_STATIC_RESOURCES>
     if ( $uri =~ /$WEB::CAPTIVE_PORTAL_STATIC_RESOURCES/o ) {
         $logger->debug("URI '$uri' (URL: $url) is a captive-portal static resource");
+        $r->set_handlers( PerlResponseHandler => ['pf::web::static'] );
```

Comment:
This is a good thing.
We should make this a configuration item so users can choose if they want to use the dynamic asset feature.